sbc_email/models/import_letters_history.py

```diff
@@ -379,28 +380,109 @@ def _get_smb_connection(self):
             return SMBConnection(
                 SmbConfig.smb_user, SmbConfig.smb_pass, 'openerp', 'nas')
 
-    @api.model
-    def import_web_letter(self):
+    @api.one
```

Comment:
Is this method called from the website ? Do you provide an id of an import from the caller or not ? If not, it is better to replace with `api.model`